tests/unit_test/tool/job/config/configer_test.py

```diff
@@ -116,3 +161,31 @@ def test_split_key(self):
             assert split_array_key("components[1.args.model.path")
         except ValueError:
             assert True
+
+    @pytest.mark.parametrize("input_file_path, expected_config_file_path", GET_CONFIG_FILE_PATH_TEST_CASES)
+    def test_get_config_file_path(self, input_file_path, expected_config_file_path):
+        job_folder = "/tmp/nvflare/job_folder"
+        config_file_path = get_config_file_path(app_name="app", input_file_path=input_file_path, job_folder=job_folder)
+        assert config_file_path == os.path.join(job_folder, expected_config_file_path)
+
+        with pytest.raises(ValueError) as excinfo:
```

Comment:
It might be better to separate this negative case out

Otherwise it is duplicated for each positive case?

But this is minor